capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -0,0 +1,76 @@
+package com.google.univiz.scorecard;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+
+/**
+ * ScorecardData is a class representing a single university/college/institution from the parsed
+ * JSON data directly from the CollegeScorecard API. The parameters come directly from the
+ * CollegeScorecard API Data Dictionary
+ * (https://collegescorecard.ed.gov/assets/CollegeScorecardDataDictionary.xlsx), which outlines the
+ * type and description of all available data elements. The parameters used here are a select few
+ * data elements the API provides.
+ */
+@GenerateTypeAdapter
+@AutoValue
+abstract class ScorecardData {
+
+  /** @return The human-readable name for the school/college/university. */
+  @SerializedName("school.name")
+  abstract String name();
+
+  /** @return The human-readable name for the city the university is located in. */
+  @SerializedName("school.city")
+  abstract String city();
+
+  /** @return An integer that flags whether the university is the main campus or not, 1 or 0. */
+  @SerializedName("school.main_campus")
+  abstract int flagMainCampus();
+
+  /**
+   * @return An integer from range(1-8) that tells the degree of urbanization for that campus, from
```

Comment:
Does this range mean 1-8 inclusive or exclusive? Consider using [this interval notation](https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals) to properly document the ranges. For example if the range includes 1 but excludes 8, you can say `range [1, 8)`.